docs/UserGuide.md

```diff
@@ -508,43 +508,40 @@ ProductiveNUS data are saved in the hard disk automatically after any command th
-
-DIAGRAM
+<br>
+You can reschedule by using the schedule command on that assignment again.<br>
 
+1. Type the schedule command into the Command Box with the index of the assignment that you want to reschedule. You can refer to the diagram below.<br>
+   ![UserGuideReschedule](images/UserGuideReschedule.PNG)
+   *Figure 20a: Index of the assignment to reschedule labelled*
 <div markdown="span" class="alert alert-success">
 **:bulb: Tip:**
 You can copy the entire command if you intend to reschedule the assignment more than once. 
-You can paste the command into the command box after step 3 and repeat step 2-3 until you have obtain your
+You can paste the command into the command box after step 2 and repeat step 1-2 until you have obtained your
  ideal schedule.
 </div>
 
-3. Press ENTER to reschedule your assignment.
-
-DIAGRAM
-
+2. Press ENTER to reschedule your assignment and your assignment will be scheduled to another time or day as shown in the diagram below.<br>
+   ![UserGuideRescheduleSuccess](images/UserGuideRescheduleSuccess.PNG)
+   *Figure 20b: New suggested time for assignment labelled*
+<br>
 **Q**: I double-clicked the productivenus.jar file but the app is not starting. What should I do?
 **A**: 
 *For Windows Users:*
-1. Go to the folder where you have downloaded and saved the productivenus.jar file.
-2. Copy the file directory of the folder. The file directory is labelled in the diagram below.
-DIAGRAM
-3. Open your command interpreter. To do so, press the Windows key on your keyboard, type in `cmd` and press ENTER.
-4. Your command interpreter will start, and the command interpreter should look something similar to the diagram below.
-<br>
-DIAGRAM
-4. In the command interpreter, type `cd ` and paste the file directory that you have copied in step 2.
-<br>For example, `cd C:\Desktop\ProductiveNUS`.
-5. Press ENTER and the file directory will be shown in the command interpreter.
-<br>
-DIAGRAM
+1. Open the folder where you have downloaded and saved the productivenus.jar file.
+2. Copy the file directory of the folder. You can refer to the labelled diagram below.<br>
+   ![UserGuideLocateDirectory](images/UserGuideLocateDirectory.PNG)
+   *Figure 21a: File directory labelled*
+3. Open your command prompt. To do so, press the Windows key on your keyboard, type in `cmd` and press ENTER.
+4. Your command prompt will start, and it should look something similar to the diagram below.<br>
+   ![UserGuideCommandPrompt](images/UserGuideCommandPrompt.PNG)
+   *Figure 21b: Interface of Command Prompt*
+4. In the command prompt, type `cd ` and paste the file directory that you have copied in step 2.
+<br>For example, `cd C:\Users\YourUserName\Desktop\ProductiveNUS`.
+5. Press ENTER and the file directory will be shown in the command prompt as shown in the diagram below.<br>
+   ![UserGuideChangeDirectory](images/UserGuideChangeDirectory.PNG)
+   *Figure 21c: After changing file directory in Command Prompt*
```

Comment:
The description looks good! However, there seems to be something wrong with the formatting :( I have attached a screenshot of the FAQ is displayed in the UG on GitHub. 

![image](https://user-images.githubusercontent.com/58943071/98027785-23152900-1e48-11eb-812e-4e4039df23d9.png)

Reply:
changed accordingly!

Reply:
Looks good! You may want to consider adding white spaces between the figure caption and the steps to improve readibility 👍 

![image](https://user-images.githubusercontent.com/58943071/98357053-8e851380-205f-11eb-87bd-d29dde8dbfb8.png)

Reply:
ok thanks!!